meteor/lib/collections/RundownLayouts.ts

```diff
@@ -63,6 +79,7 @@ export interface RundownLayoutFilterBase extends RundownLayoutElementBase {
 	label: string[] | undefined
 	tags: string[] | undefined
 	displayStyle: PieceDisplayStyle
+	dvePreviewsAsThumbnails: boolean
```

Comment:
This seems very specific. Wouldn't it just make sense to have a flag to just enable/disable thumbnails for an entire filter? Or a list of SourceLayerTypes/SourceLayers to enable/disable thumbnails on?

Reply:
Good point, I think I'll go for an entire filter for now - as that matches the use case this is being built for at TV 2.